### PR TITLE
Refactor 

### PR DESCRIPTION
**Pull request by chantal66**
- creates SRP for functions. 
- adds Api url into single file. 
- adds a class to handle Ajax requests.

### CODE REVIEW

File: lib/helpers/ajaxHelper.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const $ = require('jquery');
+const setUrl = require('../models/apiUrl').setUrl;
+
+class AjaxRequests {
```
Comment (nickedwards109) [inline lib/helpers/ajaxHelper.js:4]: Awesome refactor, @chantal66! Keeping all of the ajax requests in one file seems like a much more clear architecture that should be much easier to work with in the future.

**Comment by chantal66** [comment]
Thank you.. we still need to do delete request it gave me some trouble so I left it alone for now

File: lib/models/food.js [hunk 2]
```diff
@@ -56,17 +53,72 @@ class Food {
...
+        $('.form-food').on('submit', (event) => {
+            Food.addNewFood(event);
+        });
+    }
 
-      $('.all-foods').on('click', '.delete', (event) => {
-        const id = event.target.dataset.id;
-        Food.delete(id);
-      });
+    static deleteEvent(){
```
Comment (nickedwards109) [inline lib/models/food.js:62]: @chantal66 I like these clearly-named functions for creating, updating, and deleting foods. This is a major improvement over our previous setup of using a single defineEventHandlers() function which did lots of different things. This refactor makes everything much more explicit and readable. Nice!

**Comment by nickedwards109** [comment]
Great work, @chantal66! I think our code is way more readable after these refactors. Awesome.

**Pull request merged**